src/PitWallAcquisitionPlugin/PitWallAcquisitionPlugin/UI/ViewModels/PluginSettingsViewModel.cs

```diff
@@ -13,11 +15,13 @@ public class PluginSettingsViewModel :
         private const string VALIDATION_PERSONALKEY_LENGTH_INVALID = "Personal key length should be at least 10 character long.";
         private const string VALIDATION_PERSONALKEY_FORMAT_INVALID = "Personal should be made of alphanumerical character and \"-\", \"_\", \"@\".";
 
-        private IPitWallConfiguration _configuration;
+        private readonly IPitWallConfiguration _configuration;
+        private readonly ILiveAggregator _aggregator;
 
-        public PluginSettingsViewModel(IPitWallConfiguration configuration)
+        public PluginSettingsViewModel(IPitWallConfiguration configuration, ILiveAggregator aggregator)
```

Comment:
issue(non-blocking) : should have a configuration service to identifies what's about configuration. Should come withe the apply configuration button.